### PR TITLE
[QATM-52] [QATM-95] Bump zk version and fix steps

### PR DESCRIPTION
**Pull request by adoblas**

### CODE REVIEW

File: src/main/java/com/stratio/qa/specs/HookGSpec.java [hunk 1]
```diff
@@ -176,27 +176,11 @@ public void restClientSetup() throws Exception {
                 .build()));
     }
 
-    @After(order = 10, value = "@rest")
```
Comment (witokondoria) [inline src/main/java/com/stratio/qa/specs/HookGSpec.java]: This hook deletion is unrelated

File: src/test/resources/features/zookeeperSteps.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
+@rest
```
Comment (witokondoria) [inline src/test/resources/features/zookeeperSteps.feature]: Irrelevant tag

File: src/test/resources/features/zookeeperSteps.feature [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Feature: Zookeeper steps test
+
+  Scenario: Connection to Zk
+    Given I connect to Zookeeper at '${ZOOKEEPER_HOSTS}'
+
+  Scenario: zookeeper node does not exist
```
Comment (witokondoria) [inline src/test/resources/features/zookeeperSteps.feature]: The node exists or doesn't?

File: src/test/resources/features/zookeeperSteps.feature [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+  Scenario: create zookeeper non ephemeral node and check existance
+    Then I create the zNode '/testnonephemeral' which IS NOT ephemeral
+    Then the zNode '/testnonephemeral' exists
+
+  Scenario: create zookeeper node and check existance
+    Then I create the zNode '/testnonephemeraldata' with content 'midata' which IS ephemeral
+    Then the zNode '/testnonephemeraldata' exists and contains 'midata'
+
+  Scenario: zookeeper
```
Comment (witokondoria) [inline src/test/resources/features/zookeeperSteps.feature]: An scenario just for disconnecting?

File: src/test/resources/features/kafkaSteps.feature [hunk 5]
```diff
@@ -13,9 +8,11 @@ Feature: Cassandra steps test
 
   Scenario: Send message to kafka topic
     Given I send a message 'hello' to the kafka topic named 'testqa'
+    Then The kafka topic 'testqa' has a message containing 'hello'
 
   Scenario: Increase partitions in kafka topic
     Given I increase '1' partitions in a Kafka topic named 'testqa'
 
   Scenario: A kafka topic does not exist
```
Comment (antonioalf) [inline src/test/resources/features/kafkaSteps.feature]: Revisar este escenario. Yo lo dividiría en dos:
- Comprobar que un topic not existe
- Borrar topic y comprobar que se ha borrado

File: src/main/java/com/stratio/qa/specs/ThenGSpec.java [hunk 6]
```diff
@@ -655,7 +651,7 @@ public void kafkaTopicExist(String topic_name) throws KeeperException, Interrupt
      */
     @Then("^A kafka topic named '(.+?)' not exists")
```
Comment (antonioalf) [inline src/main/java/com/stratio/qa/specs/ThenGSpec.java]: Debería ser:
@Then("^A kafka topic named '(.+?)' does not exist")

**Comment by adoblas** [comment]
This PR also contains [QATM-95](https://stratio.atlassian.net/browse/QATM-95)
